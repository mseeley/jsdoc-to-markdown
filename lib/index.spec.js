/* eslint-env jest */

describe("index", () => {
  it("exports the expected members", () => {
    expect(Object.keys(require("./index"))).toMatchSnapshot();
  });
});
