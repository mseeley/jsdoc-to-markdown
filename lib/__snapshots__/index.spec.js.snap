// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index exports the expected members 1`] = `Array []`;
